cfme/configure/tasks.py

```diff
@@ -189,18 +123,13 @@ def is_cluster_analysis_finished(name, **kwargs):
     return is_analysis_finished(name=name, task_type='cluster', **kwargs)
 
 
-def is_task_finished(tab, page, task_name, expected_status, clear_tasks_after_success=True):
-    el = None
-    try:
-        if not sel.is_displayed(tasks_table) or not tabs.is_tab_selected(tab):
-            sel.force_navigate(page)
-        el = tasks_table.find_row_by_cells({
-            'task_name': task_name,
-            'state': expected_status
-        })
-        if el is None:
-            return False
-    except Exception:
+def is_task_finished(tab_destination, task_name, expected_status, clear_tasks_after_success=True):
```

Comment:
I went back and forth on this. I think it would open up some improvements on the  tests, where the functionality of some of these methods is duplicated. I didn't want this PR's scope to creep too much beyond the navigation updates.